### ISSUE TITLE
Library Editor: Allow to display and copy UUIDs

### ISSUE THREAD

**Issue by ubruhin** [LibrePCB/LibrePCB#348]
##### SUMMARY
UUIDs were removed from the UI in #337 to avoid confusion of users. But they are sometimes still needed, for example when creating elements by scripts (e.g. [librepcb-parts-generator](https://github.com/LibrePCB/librepcb-parts-generator)). So there should be a way to display UUIDs (and copy them to clipboard) of library elements, symbol pins, footprint pads etc.

##### IMPLEMENTATION
I'm not sure how this could be implemented without cluttering the UI. It's important that UUIDs are normally not visible.

**Comment by dbrgn** [comment]
What about a menu entry next to the "open in file manager" entry called either "developer information" or "details"? It could contain the own UUID, the category UUID, etc. And maybe even the source code.

**Comment by ubruhin** [comment]
Yeah that would be an option. But there are much more UUIDs than only the library element's UUID: Category, Component Signals, Component Symbol Variants, Component Symbol Variant Items, Package Pads, Polygons, Drills, Texts etc.

**Comment by dbrgn** [comment]
Yeah of course, you could display them in a tree like structure.

**Comment by dbrgn** [comment]
How about an entry in the context menu called "Copy UUID to Clipboard"?

![image](https://user-images.githubusercontent.com/105168/179397905-6bf846b0-3917-4c7a-a3f2-04494ec32c55.png)
